history-of-the-quakers-v3/original/20-book-5-chapter-5.adoc

```diff
@@ -213,16 +220,21 @@ by such a public vindication of your honour, as your own prudence,
 upon these suggestions, will now show you to be most necessary, and most expedient.
 I am, with unfeigned and most respectful affection,
 
+[.signed-section-closing]
 Honoured Sir,
 
-Your most humble
-
-And most obedient servant.
+[.signed-section-signature]
+Your most humble and most obedient servant.
```

Comment:
This seems too long and looks a little funny in the PDF, but idk what else to do.

Reply:
yeah, this is sorta weird. How about we nix the [ssc] above and move the `Honored Sir,` up onto the same line with the previous phrase (line 221). 

Then, I think the sscc seems pretty good, even though it's a little long. It does seem like it's his "signature" 👍